src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserController.java

```diff
@@ -100,4 +107,44 @@
 
         return profUsersEntityResponse;
     }
+
+    @ApiOperation(
+            value = "Modify roles for user",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 201,
```

Comment:
Check if It needs to be 200 status code or 201. 
You can add one more ApiResponse annotation for 400 -BAD Request